packages/frontend/components/VideoList.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Video from './Video'
+
+export default function VideoList({ videos }) {
```

Comment:
also same as above, maybe array of `video` type ?

Reply:
fixed